frontend/src/app/timezone.ts

```diff
@@ -0,0 +1,9 @@
+export interface Timezone {
```

Comment:
**question:** There is a very similar typedef in https://github.com/USDAForestService/NRM-Grants-Agreements/blob/development/api/src/index.ts#L10. For this, I'm fine repeating it here, but it has me wondering about re-use between front- and backend code in the future.

Reply:
That...is a great question and definitely worth exploring. I opened #373 to dig into this further.